pkg/kubernetes/kubernetes.go

```diff
@@ -20,16 +20,25 @@ import (
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Init deploys the Dapr operator
-func Init() error {
+const (
+	daprLatestVersion = "latest"
```

Comment:
Can this be exported? be made into `DaprLatestVersion` and then used from `init.go`:

`InitCmd.Flags().StringVarP(&runtimeVersion, "runtime-version", "", kubernetes.DaprLatestVersion, "The version of the Dapr runtime to install. for example: v0.1.0")`

Reply:
@yaron2 , if we export, then standalone package need to be changed as well. Also I prefer that it is better to keep init variable (runtimeVersion) separately and get the variable value ( default ) and then pass it over in packages and define "latest" as const in kubernetes.go which is as per current PR.

Reply:
ok